rmgpy/reactionTest.py

```diff
@@ -144,24 +144,24 @@ def setUp(self):
 
         s_h2 = Species(
             molecule=[m_h2],
-            thermo=ThermoData(Tdata=([300, 400, 500, 600, 800, 1000, 1500],
+            thermo=ThermoData(Tdata=([300, 400, 500, 600, 800, 1000, 1500, 2000],
```

Comment:
Yes, without it, the new test raises a warning because it’s unable to estimate free energy at 2000k (but still passes because we are only looking at units). Adding the extra point avoids that warning